projects/ontimize-web-ngx/src/lib/components/table/column/cell-editor/text/o-table-cell-editor-text.component.html

```diff
@@ -1,7 +1,7 @@
 <ng-template #templateref let-cellvalue="cellvalue" let-rowvalue="rowvalue">
   <div [formGroup]="formGroup">
-    <mat-form-field floatLabel="never">
-      <input #input matInput type="text" [placeholder]="getPlaceholder()" [formControl]="formControl"
+    <mat-form-field floatLabel="never" style="width:100%" cdkFocusInitial>
```

Comment:
I don't like this hardcoded _style_ attribute here. I think it should be done in the main css stylesheet.